tokio-timer/src/clock/mod.rs

```diff
@@ -16,6 +16,7 @@
 //! [`Instant::now`]: https://doc.rust-lang.org/std/time/struct.Instant.html#method.now
 //! [`with_default`]: fn.with_default.html
 
+#[allow(clippy::module_inception)]
```

Comment:
We can rename the nested `clock` module or just move the contents into `mod.rs`.

Reply:
Moved contents into mod.rs (https://github.com/tokio-rs/tokio/pull/1335/commits/f21025033a59235593e0aef8f6b9265692970636).